features/proxy-fields.feature

```diff
@@ -0,0 +1,225 @@
+      | aanschrijfwijze.aanspreekvorm | naam.adellijkeTitelPredicaat,aanduidingNaamgebruik,geslachtsaanduiding |
+      | gebruikInLopendeTekst         | naam,geslachtsaanduiding,partners.naam                                 |
+      | leeftijd                      | geboorte.datum                                                         |
+      | nationaliteiten.type          | nationaliteit,aanduidingBijzonderNederlanderschap                      |
+      | overlijden.indicatieOverleden | overlijden.datum                                                       |
+
+    @proxy
+    Abstract Scenario: met fields is gevraagd om meerdere afgeleide en/of andere gegevens <gevraagd aan proxy>
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | <gevraagd aan proxy>            |
+      Dan zoekt de proxy de persoon bij RvIG met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | <vragen aan RvIG>               |
+
+      Voorbeelden:
+      | gevraagd aan proxy                         | vragen aan RvIG                         |
+      | naam.voorletters,leeftijd                  | naam.voornamen,geboorte.datum           |
+      | burgerservicenummer,leeftijd,volledigeNaam | burgerservicenummer,geboorte.datum,naam |
+
+    @proxy
+    Abstract Scenario: met fields is gevraagd om meerdere afgeleide en andere gegevens met overlap <gevraagd aan proxy>
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | <gevraagd aan proxy>            |
+      Dan zoekt de proxy de persoon bij RvIG met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | <vragen aan RvIG>               |
+
+      Voorbeelden:
+      | gevraagd aan proxy                                                              | vragen aan RvIG                                   |
+      | aanhef,aanschrijfwijze.naam                                                     | naam,geslachtsaanduiding,partners.naam            |
+      | naam.volledigeNaam,naam.voorletters                                             | naam                                              |
```

Comment:
waarom zou je deze vertaling willen en niet gewoon 'naam,naam.voornamen'? Zeg je hiermee dat de proxy ook 'fields=naam.geslachtsnaam,naam.voornamen,naam.voorvoegsel,[alle properties van naam]' moet vertalen naar 'fields=naam'?

Reply:
de vertaling van fields=naam.volledigeNaam is fields=naam, omdat daarvoor alle naamgegevens nodig zijn. wanneer daarnaast ook voorletters wordt gevraagd, hoeft er niet meer gevraagd te worden.

De vraag is hoe belangrijk het is om dubbelingen te voorkomen in de vraag en hoe moeilijk het is om het te voorkomen. 
Zou niet heel erg hoeven te zijn, fields geeft daar als het goed is geen fouten op (bijv. fields=burgerservicenummer,burgerservicenummer geeft geen fout, maar gewoon 1x burgerservicenummer)

@KayodeBakker heeft de API er last van wanneer in fields eenzelfde veld meerdere keren wordt gevraagd? Zou dat voor systeembelasting of responsietijd gevolgen kunnen hebben? Bijvoorbeeld bij fields=naam,naam,naam.voornamen.

Reply:
mee eens